### PR TITLE
Close the recent searches on Shift+Tab

### LINKED ISSUE #1886: Recent searches combobox doesn't close when focus is lost
## Description

If a user tabs through the header navigation, the recent searches popup isn't closed while tabbing through.

## Reproduction

1. Navigate to the search page
1. Press tab until you've navigated into the search bar and the recent searches popup opens
1. Tab out of the search bar
1. Observe the popover is still open

## Screenshots

https://user-images.githubusercontent.com/6351754/194308291-b148d5fe-5dec-48c3-a169-d5cf4f4674b2.mp4

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #1886 by @zackkrida

## Description

This is a follow up for [1889](https://github.com/WordPress/openverse-frontend/pull/1889) to close the Recent searches popover when pressing <kbd>Shift</kbd>+<kbd>Tab</kbd> while inside the input (when you get to the Openverse logo).

## Testing Instructions

On main, when you press <kbd>Shift</kbd>+<kbd>Tab</kbd> while inside the search input, you focus the Openverse logo, but the Recent searches popover stays open. Here, it should close.

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VHeader/VSearchBar/VSearchBar.vue [hunk 1]
```diff
@@ -211,6 +211,11 @@ export default defineComponent({
         // If a recent search is selected, populate its value into the input.
         modelMedium.value = entries.value[selectedIdx.value]
 
+      // Hide the recent searches popover when the user presses shift+tab on the input.
+      if (key === keycodes.Tab && event.shiftKey) {
+        handleBlur()
```
Comment (dhruvkb) [inline src/components/VHeader/VSearchBar/VSearchBar.vue:216]: I'm a little bothered by the difference between this line and line 221 below.

I'm not sure if we should use `isRecentVisible.value = false` to match the code in the next if block or update that to use `handleBlur()` instead.

**Review by dhruvkb — APPROVED** [review]
This is good, except a non-blocking nit.

File: src/components/VHeader/VSearchBar/VSearchBar.vue (hunk 1 above)
Comment (obulat) [reply src/components/VHeader/VSearchBar/VSearchBar.vue:216]: I haven't noticed it before, and I'm all in for consistency! I wonder if it's best to extract a `hideRecent` function that would set `isRecentVisible.value` to `false` to make it easier to understand the code.

Comment (obulat) [reply src/components/VHeader/VSearchBar/VSearchBar.vue:216]: I've renamed `handleBlur` to `hideRecentSearches`, and `handleFocus` to `showRecentSearches` (they actually just set the `isRecentVisible.value` to `false`/`true`), @dhruvkb. I feel like that makes it easier to understand the template code.

**Review by dhruvkb — APPROVED** [review]
Nice change!

**Review by zackkrida — APPROVED** [review]
👍

**Pull request merged**